### ISSUE TITLE
Forminator Integration API update

### ISSUE THREAD

**Issue by spelkey-ucd** [UCDavisLibrary/ucdlib-awards#10]
https://github.com/UCDavisLibrary/forminator-addon-rt/issues/9

**Comment by spelkey-ucd** [comment]
might need to do db migration.

```sql
UPDATE wp_frmt_form_entry_meta
SET meta_key = REPLACE(meta_key, 'forminator_addon_ucdlib-awards_', 'forminator_addon_ucdlibawards_')
WHERE meta_key LIKE 'forminator_addon_ucdlib-awards_%';
```

**Comment by spelkey-ucd** [comment]
released v1.1.0
